distribution/ddf-common/src/main/resources/bin/ddfsolr.bat

```diff
@@ -45,7 +45,12 @@ IF "%COMMAND%"=="start" (
 
 
 IF "%COMMAND%"=="stop" (
-   SET /P STOP_KEY=<%STOPKEY_FILE%
+REM Github DDF issue 4584
+REM Suppress error message about missing STOPKEY file. The script that calls ddfsolr.bat
+REM first calls stop to shutdown any current running Solr instances on the designated port.
+REM Because stop is called before start, there will never be a STOPKEY file
+REM and in those cases the missing file message is spurious.
+   SET /P STOP_KEY=<%STOPKEY_FILE% > nul 2> nul
```

Comment:
Will this still produce an error message when stop has been called after start?

Reply:
No.